packages/flutter/test/rendering/annotated_region_test.dart

```diff
@@ -68,10 +67,22 @@ void main() {
 
     test('finds a value under a TransformLayer', () {
       final Matrix4 transform = Matrix4(
```

Comment:
Its possible to semi-preserve the old formatting by adding a comment at the end of the first line, like:

```
transform: Matrix4(
  3.0, 0.0, 0.0, 0.0, // this is a matrix bro
  0.0, 3.0, 0.0, 0.0,
  0.0, 0.0, 1.0, 0.0,
  0.0, 0.0, 0.0, 1.0,
),
```

However this does not preserve column alignment